goty-server/src/main/kotlin/com/aleinin/goty/submit/SubmissionController.kt

```diff
@@ -51,7 +58,7 @@ class SubmissionController(
                         mostAnticipated = submissionRequest.mostAnticipated,
                         bestOldGame = submissionRequest.bestOldGame,
                         enteredGiveaway = submissionRequest.enteredGiveaway,
-                        updatedOn = System.currentTimeMillis(),
+                        updatedOn = clock.millis(),
```

Comment:
Curious about your thoughts on refactoring System and Instant usage to clock @erichaagdev 

on one hand DI-ing a clock instance seems a bit much, but also it makes testing a breeze compared to messing with static mocks

Reply:
Yeah this is exactly how I would do it. Nice. 👍